tools/cxbe/Xbe.cpp

```diff
@@ -184,6 +239,10 @@ Xbe::Xbe(const char *x_szFilename)
                 goto cleanup;
             }
 
+            if (!strcmp("CXBE0", m_LibraryVersion[v].szName))
+            {
+                is_cxbe_generated = true;
+            }
```

Comment:
I don't think we should track wether this was generated by `CXBE`. This library version header was also added rather late in CXBE development.